packages/mobile/src/analytics/ValoraAnalytics.ts

```diff
@@ -0,0 +1,150 @@
+import Analytics, { Analytics as analytics } from '@segment/analytics-react-native'
+import Firebase from '@segment/analytics-react-native-firebase'
+import DeviceInfo from 'react-native-device-info'
+import { AppEvents } from 'src/analytics/Events'
+import { AnalyticsPropertiesList } from 'src/analytics/Properties'
+import { SEGMENT_API_KEY } from 'src/config'
+import { store } from 'src/redux/store'
+import Logger from 'src/utils/Logger'
+import * as Web3Utils from 'web3-utils'
+
+const TAG = 'ValoraAnalytics'
+
+async function getDeviceInfo() {
+  return {
+    AppName: DeviceInfo.getApplicationName(),
+    Brand: DeviceInfo.getBrand(),
+    BuildNumber: DeviceInfo.getBuildNumber(),
+    BundleId: DeviceInfo.getBundleId(),
+    Carrier: await DeviceInfo.getCarrier(),
```

Comment:
Can we remove this? I haven't had time to do the PII purge yet but this should probably go

Reply:
carrier is really helpful for understanding verification issues...

Reply:
Feature request for analytics: ability to log encrypted fields & decrypt on-demand while querying, such that risk is minimized if analytics infrastructure is compromised

Reply:
I'm certainly open to it but don't feel strongly either way.

Most countries only have a handful of carriers, which can be easily implied by country codes. I can see having the granularity of carrier be useful for analytics around verification but you are much closer to that than I am so I'd defer to your assessment of the tradeoffs.

Reply:
Yeah good points here. The verification debugging bit is a strong argument. I guess I'm okay with keeping it included for a while.